assignments/function-conversion.js

```diff
@@ -4,12 +4,18 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
+let myFunction = () => {console.log('Function was invoked!')};
+myFunction();
 
```

Comment:
should return the string "Function was invoked!" and the console.log the function call rather than having just a console.log in the function without a return.